src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -422,6 +422,7 @@
 helps['deployment validate'] = """
 type: command
 short-summary: Validate whether a template is valid at subscription scope.
+long-summary: Validate whether a template is valid at subscription scope. Please specify only one of --template-file FILE | --template-uri URI | --template-spec.
```

Comment:
The content of `short-summary` does not need to be included in `long-summary`. 
In addition, please tell users why they need to pass in one of the three parameters.
```suggestion
short-summary: Validate whether a template is valid at subscription scope.
long-summary: Please specify only one of --template-file FILE | --template-uri URI | --template-spec to input the ARM template.
```